pageserver/src/aux_file.rs

```diff
@@ -1,11 +1,18 @@
 use bytes::{Buf, BufMut, Bytes};
 use pageserver_api::key::{Key, AUX_KEY_PREFIX, METADATA_KEY_SIZE};
+use sha2::Digest;
 use tracing::warn;
 
+fn hash256(data: &[u8]) -> [u8; 32] {
+    let mut hasher = sha2::Sha256::new();
+    hasher.update(data);
+    hasher.finalize().into()
```

Comment:
```suggestion
    sha2::Sha256::digest(data).into()
```

Since you `use sha2::Digest` earlier, this should be enough. However, I'm unsure if this really needs an fn.

Reply:
Pros for not using this function: I think using the generic version might be able to skip creating the usual vectored versions because the size will be known. I haven't checked this.